### ISSUE TITLE
toolbar z-index is higher than temporary drawer

### ISSUE THREAD

**Issue by bernessco** [material-components/material-components-web#935]
## packages/mdc-toolbar/mdc-toolbar.scss
```
.mdc-toolbar--fixed {
--
  | position: fixed;
  | top: 0;
  | left: 0;
  | z-index: 4;
  |  
  | @include mdc-elevation(4);
  |
}
```
## packages/mdc-drawer/temporary/mdc-temporary-drawer.scss
```
.mdc-temporary-drawer {
--
  | @include mdc-drawer-base_;
  |  
  | position: fixed;
  | top: 0;
  | left: 0;
  | width: 100%;
  | height: 100%;
  | overflow: hidden;
  | pointer-events: none;
  | box-sizing: border-box;
  | contain: strict;
  | z-index: 3;
...
```

**Comment by Shyam-Chen** [comment]
![default](https://user-images.githubusercontent.com/13535256/28106827-029a235a-6718-11e7-88b0-6c8fdd9412dd.PNG)

**Comment by touficbatache** [comment]
I already submitted #925 to fix this: waiting for it to be merged. Keep this issue open in reference.

**Comment by lynnjepsen** [comment]
Merged #925 !
